runtime/lua/vim/lsp/protocol.lua

```diff
@@ -676,6 +676,19 @@ function protocol.make_client_capabilities()
         };
         hierarchicalDocumentSymbolSupport = true;
       };
+      workspaceSymbol = {
+        dynamicRegistration = false;
+        symbolKind = {
+          valueSet = (function()
+            local res = {}
+            for k in pairs(protocol.SymbolKind) do
+              if type(k) == 'number' then table.insert(res, k) end
+            end
+            return res
+          end)();
+        };
+        hierarchicalWorkspaceSymbolSupport = false;
```

Comment:
I've fixed wrong handling hierarchical symbols with #12222 .
So we can set this true.